api/v1alpha1/openapi.yaml

```diff
@@ -280,6 +302,10 @@ components:
           format: date-time
         sshKey:
```

Comment:
Let's remove sshkey? Should be now part of agent.

Reply:
I wanted to but it breaks the generation of ova I think.